### PR TITLE
Bug/nav mesh auto generation error

### PR DESCRIPTION
**Pull request by RogerNogue**
fixed the bug

### CODE REVIEW

File: Source/ModuleNavigation.cpp [hunk 1]
```diff
@@ -79,10 +79,10 @@ void ModuleNavigation::sceneLoaded(JSON * config)
 
 	meshGenerated = nav->GetUint("Generated", false);
 	drawNavMesh = nav->GetUint("DrawNavMesh", true);
-	if(meshGenerated)
+	if (!meshGenerated) return;
 	renderMesh = nav->GetUint("RenderNavMesh", false);
 	numObjects = nav->GetInt("numObjects");
-
+	cyclesToWaitWhenLoaded = 5;
```
Comment (AlbertVVila) [inline Source/ModuleNavigation.cpp]: WTF

File: Source/ModuleNavigation.cpp [hunk 2]
```diff
@@ -174,6 +174,11 @@ update_status ModuleNavigation::Update(float dt)
 {
 	if (autoNavGeneration)
 	{
+		if (cyclesToWaitWhenLoaded > 0)
```
Comment (AlbertVVila) [inline Source/ModuleNavigation.cpp]: WTF. Why?

**Comment by AlfonsoMestres** [comment]
This PR is just for the meeting on thursday 23/05 to avoid Game issues when doing the DEMO.
Right now we have this implemented in ShowDemo branch and should be fixed ASAP.

**Comment by RogerNogue** [comment]
It was a quick fix to unblock designers' work

**Pull request closed without merging**